iOS-SDKTests/iOS_SDKTests.swift

```diff
@@ -35,13 +35,21 @@ class iOS_SDKTests: XCTestCase {
     }
     
     func testDateLocalization() {
-        //Set the expected result in format # days, using the app settings localization
-        let expectedResult: String = "5 dagen"
+        //Check the device language
+        let languageSetting = SettingsManager.get(type: .language)
+        guard languageSetting.code == "en" else {
+            XCTFail("The phone language isn't in english")
+            return
+        }
+        
+        //Set the expected result in format # days, for the test the language setting is expected in english
+        let expectedResult: String = "5 days"
+        
         //Set in seconds the time in the past for the date to compare. Example 3,600 == 1 hour, 360,000 == 100 hours
-        let secondsInThePast: Double = 432000 //This number should be seconds and in positive
+        let secondsInThePast: Double = 432001 //This number should be seconds and in positive
```

Comment:
Why is it one second more now?

Reply:
Depending on the time my mac took to run the app, sometimes I was getting the result as 4 days instead of 5, the actual result was 4.999.. and so on, the extra second avoids this problem.